### ISSUE TITLE
Microsoft popular system tools are not available to install e.g. TCPview.

### ISSUE THREAD

**Issue by salarmehr** [microsoft/winget-cli#1268]
I'm new to this. I tried to install [TCP](https://docs.microsoft.com/en-us/sysinternals/downloads/tcpview) view through Winget, and couldn't. It seems https://docs.microsoft.com/en-us/sysinternals/downloads/ tools are not available yet. Considering most of them do not have a native installer (they are just .eve files) they are the perfect candidate to be installed by winget.

**Comment by ItzLevvie** [comment]
Blocked by https://github.com/microsoft/winget-cli/issues/140; see the Sysinternals developer's response at https://github.com/microsoft/winget-cli/issues/140#issuecomment-628917702.

**Comment by denelon** [comment]
@salarmehr, is this ask a duplicate of #140 specific to your desire to have TCPview or something else?

**Comment by salarmehr** [comment]
@denelon so in this ticket I raised a request to add Sysinternals tools to the repositories in any possible way. 

Not exactly a duplicate of #140. But this can be close if you think resolving #140 can bring Sysinternals tools automatically?

**Comment by denelon** [comment]
@salarmehr I haven't attempted to run validation against Sysinternals as there is no installer yet. I'm expecting at least one of if not several of the individual tools to trigger a PUA warning from at least one of the scans (either static or dynamic). If that happens, we are not able to approve due to policy. As soon as we support .zip #140 or they produce an installer, I'm expecting it will be submitted.

**Comment by denelon** [comment]
Now that Sysinternals has an installer and WinGet has support for portable packages and .zip-based packages, I'm going to close this issue.

**Comment by mdanish-kh** [comment]
https://github.com/microsoft/winget-pkgs/tree/master/manifests/m/Microsoft/Sysinternals/TCPView

TCPView is available now through WinGet. You can install it using `winget install tcpview`. 

If you would like to see other sysinternals tools in WinGet, I'll encourage you to open package request issues over at [WinGet Community Packages Repository](https://github.com/microsoft/winget-pkgs/issues), or even better, you can add them yourself by following the [Contributor's Guide](https://github.com/microsoft/winget-pkgs/blob/master/CONTRIBUTING.md)

**Comment by o-l-a-v** [comment]
There's also the Microsoft Store package containing all the SysInternals tools.

* https://apps.microsoft.com/store/detail/9P7KNL5RWT25?hl=en-us&gl=us

Can be installed with winget:

`winget install --source msstore --id 9P7KNL5RWT25`.

**Comment by denelon** [comment]
The Sysinternals suite is now available via WinGet, and we have support for discrete portable packages.

TCPview is there for example:
https://github.com/microsoft/winget-pkgs/tree/master/manifests/m/Microsoft/Sysinternals/TCPView/4.19
